src/constants/gameplay.asm

```diff
@@ -339,6 +339,30 @@ TRADING_ITEM_NECKLACE          equ $0C
 TRADING_ITEM_SCALE             equ $0D
 TRADING_ITEM_MAGNIFIYING_GLASS equ $0E
 
+; Values for hRoomStatus and wOverworldRoomStatus
+;
+; Flags can be combined. For example, visiting the first dungeon's screen (80)
+; and opening it with the key (10) would put that byte at 90.
+ROOM_STATUS_NONE            equ $00
+ROOM_STATUS_DOOR_OPEN_RIGHT equ $01
+ROOM_STATUS_DOOR_OPEN_LEFT  equ $02
+ROOM_STATUS_DOOR_OPEN_UP    equ $04
+ROOM_STATUS_DOOR_OPEN_DOWN  equ $08
+ROOM_STATUS_EVENT_1         equ $10
+ROOM_STATUS_EVENT_2         equ $20
+ROOM_STATUS_EVENT_3         equ $40
```

Comment:
Main use of this flag seems to be the keyblocks in dungeons, but a few "scripted" events seem to use it as well on the overworld.

Reply:
One day I'll probably split better the status room flags for overworld (where they have often a precise use) from indoors (where it seems every entity and it's cousin use them for different reasons).